src/QsCompiler/CSharpGeneration/EntryPoint.fs

```diff
@@ -397,7 +397,6 @@ let private driverSettings context =
         namedArg "quantumSimulatorName" <| literal AssemblyConstants.QuantumSimulator
         namedArg "sparseSimulatorName" <| literal AssemblyConstants.SparseSimulator
         namedArg "toffoliSimulatorName" <| literal AssemblyConstants.ToffoliSimulator
-        namedArg "resourcesEstimatorName" <| literal AssemblyConstants.ResourcesEstimator
```

Comment:
@DmitryVasilevsky, @msoeken,
Please make sure this PR removes all the deprecation messages added in [here](https://github.com/microsoft/iqsharp/pull/766/files).